test/groovy/src/test/java/org/mvcspec/ozark/test/groovy/GroovyIT.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright © 2017 Ivar Grimstad (ivar.grimstad@gmail.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.mvcspec.ozark.test.groovy;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.Iterator;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.gargoylesoftware.htmlunit.WebClient;
+import com.gargoylesoftware.htmlunit.html.HtmlElement;
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
+
+
+/**
+ * Unit test for simple App.
+ */
+public class GroovyIT { 
+
+    private String webUrl;
+    private WebClient webClient;
+
+    @Before
+    public void setUp() {
+        webUrl = System.getProperty("integration.url");
+        webClient = new WebClient();
+    }
+
+    @After
+    public void tearDown() {
+        webClient.closeAllWindows();
+    }
+
+    @Test
+    public void testView1() throws Exception {
+        final HtmlPage page = webClient.getPage(webUrl + "resources/hello?user=mvc");
+        final Iterator<HtmlElement> it = page.getDocumentElement().getHtmlElementsByTagName("h1").iterator();
+        assertTrue(it.next().asText().contains("mvc"));
```

Comment:
Usually `assertTrue()` assertions aren't a good choice, because error messages aren't very expressive in case of failures. They will only tell you something like "You expected true but it was false". And this won't point you to the root cause of the failure.

Instead try using something like:

```java
assertEquals("Hello mvc", it.next.asText().trim());
```

Or:

```java
assertThat(it.next.asText(), cOREMatchers.containsString("Hello mvc"));
```

This will give you errors like: "Expected 'Hello mvc' but got 'Hello $name'", which is much better to identify the root cause.

Reply:
Most of the tests are with this assertTrue, do you recommend switching to assertEquals or assertThat?